src/Page/Api.elm

```diff
@@ -48,11 +48,20 @@ getApiServerUrl { clientUrl } =
 
 changelog : List News
 changelog =
-    [ { date = "20 mars 2023"
+    [ { date = "12 avril 2023"
```

Comment:
Why replace the previous changelog entry instead of adding a new one?

Reply:
Wow I was confused with the textile PR I guess. Alzheimer is at the corner…